src/terminal/adapter/adaptDispatch.hpp

```diff
@@ -96,8 +96,10 @@ namespace Microsoft::Console::VirtualTerminal
         virtual bool SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle); // DECSCUSR
         virtual bool SetCursorColor(const COLORREF cursorColor);
 
-        virtual bool SetColorTableEntry(const size_t tableIndex,
-                                        const DWORD dwColor); // OscColorTable
+        virtual bool SetColorTableEntry(const size_t tableIndex, const DWORD dwColor); // OscColorTable
+        virtual bool SetDefaultForeground(const DWORD /*dwColor*/); // OSCDefaultForeground

```

Comment:
Why comment the params? None of the rest of the declarations in this file did that.

Reply:
Oh, maybe a copy from the other interface below... still... probably should match file consistency and file another issue to update the overall project style to be more consistent between files.

Reply:
Oh, yeah ... this is another one of those: "it's been 12 years since I worked in C++" things: if you don't know what you're doing, copy/paste carefully. I will make these match this file.